app/src/userpages/components/StreamPage/Show/KeyView/index.jsx

```diff
@@ -53,6 +54,16 @@ export class KeyView extends Component<Props> {
         }
     })
 
+    editStreamResourceKey = (streamId: StreamId, keyId: ResourceKeyId, keyName: string, keyPermission: ResourcePermission): Promise<void> =>
+        new Promise((resolve, reject) => {
+            if (this.props.streamId && keyPermission) {
+                this.props.editStreamResourceKey(streamId, keyId, keyName, keyPermission)
```

Comment:
This seems unnecessary, `this.props.editStreamResourceKey` could be called directly?